clients/da-vinci-client/src/main/java/com/linkedin/davinci/ingestion/DefaultIngestionBackend.java

```diff
@@ -61,18 +61,22 @@ public void startConsumption(VeniceStoreVersionConfig storeConfig, int partition
         Utils.waitStoreVersionOrThrow(storeVersion, getStoreIngestionService().getMetadataRepo());
     Supplier<StoreVersionState> svsSupplier = () -> storageMetadataService.getStoreVersionState(storeVersion);
     syncStoreVersionConfig(storeAndVersion.getFirst(), storeConfig);
-    AbstractStorageEngine storageEngine = storageService.openStoreForNewPartition(storeConfig, partition, svsSupplier);
-    topicStorageEngineReferenceMap.compute(storeVersion, (key, storageEngineAtomicReference) -> {
-      if (storageEngineAtomicReference != null) {
-        storageEngineAtomicReference.set(storageEngine);
-      }
-      return storageEngineAtomicReference;
-    });
 
+    long startTime = System.currentTimeMillis();
     CompletionStage<Void> bootstrapFuture =
         bootstrapFromBlobs(storeAndVersion.getFirst(), storeAndVersion.getSecond().getNumber(), partition);
 
     bootstrapFuture.whenComplete((result, throwable) -> {
+      long endTime = System.currentTimeMillis();
+      LOGGER.info("Bootstrapping duration: {} ms", endTime - startTime);
```

Comment:
If we are going to log the blob transfer time, can we also add the store name and version too otherwise it's hard to differentiate

Reply:
Also, let's call it blob transfer duration

Reply:
will do